packages/openapi-to-graphql/src/resolver_builder.ts

```diff
@@ -111,14 +111,6 @@ export function getResolver({
       resolveData.usedParams = {}
     }
 
-    /**
```

Comment:
I think this is leftover code from `subOperations`.

Reply:
Good to go, then.